### PR TITLE
Fix parsing URLs from item comments, add -p / --page flag to cmdline

### PR DESCRIPTION
**Pull request by snejus**
### Fixed

- (#56) Support parsing URLs that do not end with **.com** in item comments when importing music that was bought on Bandcamp.

### Added

- Add a new flag to the command line application for searching Bandcamp:
  **`[-p PAGE, --page PAGE]`** to enable seeing further search results

### CODE REVIEW

File: tests/test_plugin.py [hunk 1]
```diff
@@ -26,6 +26,18 @@ def check_album(actual, expected):
     assert vars(actual) == vars(expected)
 
 
+@pytest.mark.parametrize(
+    "comments, expected_url",
+    [
+        ("Visit https://label.bandcamp.com", "https://label.bandcamp.com"),
+        ("Visit https://supercommuter.net", "https://supercommuter.net"),
+        ("Visit https://no-top-level-domain", None),
```
Comment (ncraike) [inline tests/test_plugin.py:34]: Hey, sorry I'm a little late to this, but is it worth adding a test case for a URL with more than two domain parts, e.g. a sub-domain of a country TLD like `https://myband.com.au/` or `https://myband.co.uk/`? I'm not sure if your regular expression handles more than one dot in the domain portion.

Comment (snejus) [reply tests/test_plugin.py:34]: To be honest I did not even give a chance for anyone to have a look at this 😅 

This is a good idea, of course! Would you like to contribute yourself by any chance?

Comment (ncraike) [reply tests/test_plugin.py:34]: I'll see if I can find time over the next couple days. Otherwise, feel free to give it a go yourself.

Comment (ncraike) [reply tests/test_plugin.py:34]: Also, I am in Australia, so I think we're battling timezones a little. 😅

Comment (snejus) [reply tests/test_plugin.py:34]: I checked the URLs that you provided and it seems to handle them well, but I'm excited to see whether you could find a URL that breaks it!

**Pull request merged**